packages/components/src/components/vertical-tab/vertical-tab.tsx

```diff
@@ -160,7 +160,7 @@ const VerticalTab = ({
                         is_routed={is_routed}
                         header_title={header_title}
                     />
-                    {is_floating && tab_headers_note && list[curr_tab_index].has_side_note && (
+                    {is_floating && tab_headers_note && list[curr_tab_index]?.has_side_note && (
```

Comment:
Found an error when we switched from real account to demo account from any p2p page as `has_side_note` was "undefined", so added ? here to fix that. `This issue is also in staging and prod`

Reply:
https://github.com/ameerul-deriv/deriv-app/assets/103412909/8e168993-87f8-4590-a15c-0704aacf0d94